spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -896,6 +921,11 @@ private void commitIfNecessary() {
 			}
 		}
 
+		@Override
+		public void seek(String topic, int partition, long offset) {
+			this.seeks.add(new TopicPartitionInitialOffset(topic, partition, offset));
```

Comment:
I wonder why we can't reach the same via just top-level container operation.
Moreover expose it to JMX.
What am I missing?
Thanks